sdk/textanalytics/Azure.AI.TextAnalytics/samples/README.md

```diff
@@ -6,6 +6,7 @@ products:
 - azure
 - azure-cognitive-services
 - azure-text-analytics
+- language-service
```

Comment:
what is the difference between this and `azure-language-service` ? I remember @deyaaeldeen added the later one and is the one used in JS https://github.com/deyaaeldeen/azure-sdk-for-js/blob/5e57c9010d8273db4e86805c14eea9bd98033115/sdk/textanalytics/ai-text-analytics/package.json#L52